### DISCUSSION TITLE
Food Effect?

### DISCUSSION THREAD

**Discussion by DonkeySaddle** [Open-Systems-Pharmacology/Forum#1652] [Questions & Problems] [answered]
I have poked around a bit and have a rough understanding of how food affects pk-sim models, but I have been unable to find the actual equations incorporating the affect and thus have been unable to really understand what's going on. 

I tried looking at the Matlab equations and couldn't find it, but maybe I missed something?

Any suggestions would be greatly appreciated. Thanks.

DS

**Comment by AndreDlm** [comment, marked as answer]
Basically, the food intake specified in the `Events` building block triggers changes in changes in gastric pH (increases to 5.5 then decays exponentially), gastric emptying time (prolonged), and stomach lumen volume (increased by the fluid volume of the ingested meal).

You can see these equations when you export your model to MoBi and navigate to the `Events` building block under `Meal_StartEvent` in the section of your meal intake. See e.g. [here](https://github.com/Open-Systems-Pharmacology/Forum/discussions/787#discussioncomment-710500) 

See also this discussion [here](https://github.com/Open-Systems-Pharmacology/Forum/discussions/787) which might help you.

**Reply by DonkeySaddle** [reply]
Thank you so much @AndreDlm! 

I will will take some time to try and understand the discussions you have provided links for. Really appreciate it. Hopefully this will clear things up for me.
